Content/default/src/Client/vite.config.ts

```diff
@@ -0,0 +1,28 @@
+import { defineConfig } from "vite";
+import react from "@vitejs/plugin-react";
+
+const proxyPort = process.env.SERVER_PROXY_PORT || "5000";
+const proxyTarget = "http://localhost:" + proxyPort;
+
+// https://vitejs.dev/config/
+export default defineConfig({
+    plugins: [react()],
+    build: {
+        outDir: "../../deploy/public",
+    },
+    server: {
+        port: 8080,
+        proxy: {
+            // redirect requests that start with /api/ to the server on port 5000
+            "/api/": {
+                target: proxyTarget,
+                changeOrigin: true,
+            },
+            // redirect websocket requests that start with /socket/ to the server on the port 5000
+            "/socket/": {
```

Comment:
Had a discussion with @theprash and it might be best just to remove the web socket config here. We are not using any web socket functionality in the base templates so adding it back as a step in a recipe feels more consistent to everything else we do. What do you think? @mattgallagher92